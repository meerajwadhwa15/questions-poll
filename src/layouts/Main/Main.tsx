import React, { FunctionComponent, ReactNode } from "react";
import Header from "@/components/Header";
import "./Main.style.scss";

export type Props = {
  children: ReactNode;
};

const Main: FunctionComponent<Props> = ({ children }) => (
  <>
    <Header />
    <main className="container">{children}</main>
  </>
);

export default Main;
