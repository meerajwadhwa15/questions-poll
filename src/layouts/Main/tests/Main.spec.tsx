import React from "react";
import Main from "../Main";
import { create } from "react-test-renderer";

const props = {
};

test("Matches the snapshot with Main", () => {
  const component = create(
      <Main {...props}>
        <div className="test">
            Test Element
        </div>
      </Main>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
