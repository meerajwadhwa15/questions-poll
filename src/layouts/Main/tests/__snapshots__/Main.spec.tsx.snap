// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot with Main 1`] = `
Array [
  <header>
    <h1>
      Question Poll
    </h1>
  </header>,
  <main
    className="container"
  >
    <div
      className="test"
    >
      Test Element
    </div>
  </main>,
]
`;
